telos-frontend/src/components/steam/steam.jsx

```diff
@@ -42,19 +49,28 @@ const Steam = () => {
       getSteamData(steamName);
     }
   }, []);
+  const NoDataColumn = [
+    { field: 'StatusNow', headerName: `Status: ${dataMessage}`, width: 690, sortable: false },
+  ];
+
+  const NoDataRows = [
+    { id: 'Requirements1', StatusNow: '1. Steam profile is public', sortable: false },
+    { id: 'Requirements2', StatusNow: '2. Add steam key in steam.js', sortable: false },
+    { id: 'Requirements3', StatusNow: '3. Add steam name in steam.jsx', sortable: false },
+    { id: 'MoreInfo', StatusNow: 'Link below for more information:', sortable: false },
+    {
+      id: 'LinkToHelp',
+      StatusNow: 'https://github.com/UOA-SE701-Group3-2021/3Lancers/wiki/Steam-Widget',
```

Comment:
```suggestion
    { id: 'Requirements1', StatusNow: 'Looks like somthing went wrong. Try the following:', sortable: false },
    { id: 'Requirements2', StatusNow: '1. Make sure the steam profile you are searching is public', sortable: false },
    { id: 'Requirements3', StatusNow: '2. Create a bug report here: ', sortable: false },
    { 
      id: 'BugReport', 
      StatusNow: 'https://github.com/UOA-SE701-Group3-2021/3Lancers/issues/new?assignees=&labels=&template=bug_report.md&title=', 
      sortable: false 
    },
    { id: 'MoreInfo', StatusNow: 'For Contributors: Link below for more information:', sortable: false },
    {
      id: 'LinkToHelp',
      StatusNow: 'https://github.com/UOA-SE701-Group3-2021/3Lancers/wiki/Steam-Widget',
```
I think the styling looks good but the only issue is that the message is very developer-focused, where we should be focusing on the user experience. I've made a suggestion for a more user-focused message.

Reply:
True that does seem more user friendly. 
![image](https://user-images.githubusercontent.com/52488862/114806713-4d0c4780-9df9-11eb-884e-8192578c39a1.png)

Feel free to upload any code you think is best. :)